src/details/MovieDetailsCard.js

```diff
@@ -1,18 +1,55 @@
-import React from 'react'
+import React, { Component } from 'react';
 import './MovieDetailsCard.css'
 
-const MovieDetailsCard = ({movie, displayMovieLibrary}) => {
-  console.log(movie)
-  return (
-    <div className='movie-details-card' style={{backgroundImage: `url(${movie.backdrop_path})`}}>
-      <div className='movie-details-text'>
-        <button className='close-button' onClick={() => displayMovieLibrary()}>X</button>
-        <h2>{movie.title}</h2>
-        <p>RELEASE DATE: {movie.release_date}</p>
-        <p>AVERAGE RATING: {movie.average_rating.toFixed(2)}</p>
+class MovieDetailsCard extends Component {
```

Comment:
I'm guessing the idea here was to switch this to a class component to hold the state of each individual movie, since they have more details than the home list view does. Very cool!